server/org.eclipse.emfcloud.ecore.backend-app/org.eclipse.emfcloud.ecore.backend.app/src/org/eclipse/emfcloud/ecore/backend/app/CreateEcoreApplication.java

```diff
@@ -0,0 +1,49 @@
+package org.eclipse.emfcloud.ecore.backend.app;
+
+import org.eclipse.equinox.app.IApplication;
+import org.eclipse.equinox.app.IApplicationContext;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.EPackage;
+import org.eclipse.emf.ecore.EcoreFactory;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class CreateEcoreApplication implements IApplication {
+
+	@Override
+	public Object start(IApplicationContext context) throws Exception {
+        String[] args = getArgs(context);
+        createEcore(args[0], args[1], args[2], Paths.get(args[3], args[0]+".ecore"));
```

Comment:
It would be nice to check the arguments before passing them like this, to get proper error reporting (Rather than "IndexOutOfBounds" exceptions)